client/reader/follow-button/index.jsx

```diff
@@ -16,9 +16,8 @@ const ReaderFollowButton = React.createClass( {
 	mixins: [ PureRenderMixin ],
 
 	propTypes: {
-		following: React.PropTypes.bool.isRequired,
```

Comment:
Yeah. It's not the nicest. `isButtonOnly` came along because we wanted just the button in Manage Following, but with the same stats fired.